test/compilable/depsOutput21238_foo.d

```diff
@@ -0,0 +1,35 @@
+/*
+PERMUTE_ARGS:
+REQUIRED_ARGS: -deps -Jcompilable/imports
+EXTRA_SOURCES: imports/depsOutput21238_bar.d imports/depsOutput21238_baz_moduleimport.d imports/depsOutput21238_baz_stringimport.d imports/depsOutput21238_qux.d
+TRANSFORM_OUTPUT: remove_lines("druntime")
+TEST_OUTPUT:
+---
+depsImport depsOutput21238_foo (compilable$?:windows=\\|/$depsOutput21238_foo.d) : private : imports.depsOutput21238_baz_moduleimport (compilable$?:windows=\\|/$imports$?:windows=\\|/$depsOutput21238_baz_moduleimport.d)
```

Comment:
CC @MoonlightSentinel we should really find a nicer way than this :/

Reply:
How about `$p:<path>$`? I've pushed a commit.

Reply:
> How about `$p:<path>$`? I've pushed a commit.

Doesn't work. Apparently, Windows DMD prints double backslashes, but `$p:<path>$` expects singles. Back to `$?:windows=\\|/$`.